tests/mlruns/0/meta.yaml

```diff
@@ -0,0 +1,4 @@
+artifact_location: file:///Users/makoto-hiramatsu/work/github.com/himkt/optuna/tests/mlruns/0
```

Comment:
Do we need this file? I think it is not referred by any other files in this change.

Reply:
Thank you. I included this change unexpectedly. I remove it by overwriting the commit.

Reply:
Thank you.